isis/src/control/apps/jigsaw/jigsaw.xml

```diff
@@ -24,53 +25,371 @@
+      working with SAR imagery. SAR imagery also tends to have long duration images which can
+      necessitate using the <b>OVERHERMITE</b>, <b>SPKDEGREE</b>, and <b>SPKSOLVEDEGREE</b> parameters.
+    </p>
+
+    <h4>Parameter Sigmas</h4>
+    <p>
+      The initial, or a-priori, camera parameters and control points all contain error, but it is not equally
+      distributed. The <b>jigsaw</b> application allows users to input a-priori sigmas for different parameters
+      that describe how the initial error is distributed. The parameter sigmas are also sometimes
+      called uncertainties. Parameters with higher a-priori sigma values
+      are allowed to vary more freely than those with lower a-priori sigma values. The a-priori sigma values
+      are not hard constraints that limit how much parameters can change, but rather relative accuracy estimates
+      that adjust how much parameters change relative to each other.
+    </p>
+    <p>
+      The <b>POINT_LATITUDE_SIGMA</b>, <b>POINT_LONGITUDE_SIGMA</b>, and <b>POINT_RADIUS_SIGMA</b>
+      parameters set the a-priori sigmas for control points <i>that do not already have a ground source</i>.
+      If a control point already has a ground source, such as a basemap or DEM, then it already has
+      internal sigma values associated with it or is held completely fixed. The exact behavior depends
+      on how the control points were created, a topic outside the scope of this documentation.
+      These sigmas are in meters on the ground, so the latitude
+      and longitude sigmas are uncertainty in meters in the latitude and longitude directions. Generally, you will
+      choose values for these parameters based on the resolution of the images and a visual comparison with
+      other datasets. The radius sigma is also usually higher than the latitude and longitude sigmas
+      because there is less variation of the viewing geometry with respect to the radius.
+    </p>
+    <p>
+      The <b>CAMERA_ANGLES_SIGMA</b>, <b>CAMERA_ANGULAR_VELOCITY_SIGMA</b>, and
+      <b>CAMERA_ANGULAR_ACCELERATION_SIGMA</b> parameters set the a-priori sigmas for the instrument pointing.
+      Similarly, the <b>SPACECRAFT_POSITION_SIGMA</b>, <b>SPACECRAFT_VELOCITY_SIGMA</b>,
+      and <b>SPACECRAFT_ACCELERATION_SIGMA</b> parameters set the a-priori sigmas for the instrument position.
+      The sigma for each parameter is only used if you are solving for that parameter. For example,
+      if you are solving for angular velocities and not solving for any adjustments to the instrument
+      position you would enter <b>CAMERA_ANGLES_SIGMA</b> and <b>CAMERA_ANGULAR_VELOCITY_SIGMA</b>.
+      Finding initial values for these sigmas is challenging as SPICE does not have a way to specify them.
+      Some teams will publish uncertainties for their SPK and CK solutions in separate documentation or
+      the kernel comments, the commnt utility will read the comments on a kernel file. The velocity
```

Comment:
'comment' not 'commnt'

Reply:
This is correct, the naif utility is called commnt https://naif.jpl.nasa.gov/pub/naif/toolkit_docs/C/ug/commnt.html

I put a link in here so that users can read more about how to use it.